### PR TITLE
ppsspp 1.10.1

### PR DESCRIPTION
**Pull request by SeekingMeaning**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/ppsspp.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Ppsspp < Formula
   desc "PlayStation Portable emulator"
   homepage "https://ppsspp.org/"
   url "https://github.com/hrydgard/ppsspp.git",
-      :tag      => "v1.10",
-      :revision => "401df203844465b84015d4710c6f02914b890589"
+      :tag      => "v1.10.1",
```
Comment (chenrui333) [inline Formula/ppsspp.rb:5]: this is not the latest release though
https://github.com/hrydgard/ppsspp/releases

Comment (SeekingMeaning) [reply Formula/ppsspp.rb:5]: It seems they stopped marking minor version tags as releases starting with 1.7.2

Comment (SMillerDev) [reply Formula/ppsspp.rb:5]: Could you ask if this is intentional?

Comment (SeekingMeaning) [reply Formula/ppsspp.rb:5]: hrydgard/ppsspp#13101 (Also asked on their Discord server)

Comment (SeekingMeaning) [reply Formula/ppsspp.rb:5]: From a moderator in the #coding_development channel:

> yes, but v1.10.2 is probably coming soonish (it mostly affects Android, but there's a fix that affects macOS too)

**Comment by SeekingMeaning** [comment]
Superseded by #57634

**Pull request closed without merging**